content/CORE/UIReference/TopMenu/Alerts.md

```diff
@@ -1,6 +1,10 @@
 ---
 title: "Alert Notifications"
+description: "Use the Alert Notifications screen to review alerts and related options on your TrueNAS"
```

Comment:
```suggestion
description: "The **Alert Notifications** panel displays system alerts and provides options to dismiss or reopen dismissed alerts on your TrueNAS"
```

Reply:
"The Alert Notifications panel displays system alerts and provides options to dismiss or reopen dismissed alerts on your TrueNAS." As indicated above but without bold since the description is  within quotes and will be rendered in the child article listing literally.